packages/hardhat/test/e2e/crab-Migration.ts

```diff
@@ -419,9 +430,11 @@ describe("Crab Migration", function () {
         })
 
         it("Should fail if _ethToBorrow or _withdrawMaxEthToPay is passed as 0", async () => {
+            await decreaseCR1(ethers.utils.parseEther('10'))
```

Comment:
Was working before because of rounding. This will make sure CR1 will always be less as this test expects cr1 to be less